bikeshare_app/templates/checkin.html

```diff
@@ -0,0 +1,115 @@
+        <div class="ui piled segment">
+            <a id="registerHeader" href="{{ url_for('index') }}">
+                <h2 class="ui logo header">
+                    <img src="{{ url_for('static', filename='images/logo.png') }}">
+                    <div class="content">
+                        How was your ride?
+                    </div>
+                </h2>
+            </a>
+        </div>
+        <form class="ui large form">
+            <h2 class="ui horizontal divider header">
+                <i class="check square icon"></i>
+                Does the bike have any damages?
+            </h2>
+
+            <div>
+                <div class="inline fields">
+                    <div class="field">
+                        <div class="ui radio checkbox">
+                            <input type="radio" id="damageYes" name="standing" tabindex="0" value="Yes" class="hidden">
+                            <label>Yes</label>
+                        </div>
+                    </div>
+                    <div class="field">
+                        <div class="ui radio checkbox">
+                            <input type="radio" id="damageNo" name="standing" tabindex="0" value="No" class="hidden">
+                            <label>No</label>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="field" name="damageTextArea">
+                    <label>Explanation:</label>
+                    <textarea></textarea>
+                </div>
+            </div>
+
+            <h2 class="ui horizontal divider header">
+                <i class="lock icon"></i>
+                Was the bike locked up?
+            </h2>
+
+            <div class="inline fields">
+                <div class="field">
+                    <div class="ui radio checkbox">
+                        <input type="radio" name="location" tabindex="0" class="hidden">
+                        <label>Yes</label>
+                    </div>
+                </div>
+                <div class="field">
+                    <div class="ui radio checkbox">
+                        <input type="radio" name="location" tabindex="0" class="hidden">
+                        <label>No</label>
+                    </div>
+                </div>
+            </div>
+
+            <h2 class="ui horizontal divider header">
+                <i class="privacy icon"></i>
+                Was the key returned?
+            </h2>
+
+            <div class="inline fields">
+                <div class="field">
+                    <div class="ui radio checkbox">
+                        <input type="radio" name="location" tabindex="0" class="hidden">
+                        <label>Yes</label>
+                    </div>
+                </div>
+                <div class="field">
+                    <div class="ui radio checkbox">
+                        <input type="radio" name="location" tabindex="0" class="hidden">
```

Comment:
Name attribute should reflect the question at hand and not be a copy-paste from another existing file.

This in particular breaks because the radio button will allow for only one click among all marked as `name="location"`. You won't be able to answer either question correctly because they are incorrectly grouped together.